website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -29,17 +35,117 @@ saved state on startup if the state
 
 This bug is fixed in Consul versions 1.13.1 and newer.
 
-### gRPC TLS
-
-In prior Consul versions if HTTPS was enabled for the client API and exposed
-via `ports { https = NUMBER }` then the same TLS material was used to encrypt
-the gRPC port used for xDS. Now this is decoupled and activating TLS on the
-gRPC endpoint is controlled solely with the gRPC section of the new 
-[`tls` stanza](/docs/agent/config/config-files#tls-configuration-reference).
-
-If you have not yet switched to the new `tls` stanza and were NOT using HTTPS
-for the API then updating to Consul 1.13 will activate TLS for gRPC since the
-deprecated TLS settings are used as defaults.
+#### If using auto-encrypt or auto-config
+
+**Do not upgrade to Consul 1.13 yet** if using
+[auto-encrypt](/docs/agent/config/config-files#auto_encrypt) or
+[auto-config](/docs/agent/config/config-files#auto_config).
+
+In Consul 1.13, auto-encrypt and auto-config both cause Consul
+to require gRPC communication with Envoy proxies to now use TLS,
+but the Envoy proxies are unlikely to be configured to use gRPC.
+This will cause Envoy proxies (sidecars, gateways) to disconnect
+from the control plane (Consul agents).
```

Comment:
```suggestion
In Consul 1.13, auto-encrypt and auto-config both cause Consul
to require TLS for gRPC communication with Envoy proxies. However,
since the Envoy proxies are unlikely to be configured to use gRPC,
this will cause Envoy proxies (sidecars, gateways) to disconnect
from the control plane (Consul agents).
```

Reply:
```suggestion
In Consul 1.13, auto-encrypt and auto-config both cause Consul
to require gRPC communication with Envoy proxies to now use TLS.
In environments where the Envoy proxies are not configured to use TLS,
this will cause Envoy proxies (sidecars, gateways) to disconnect
from the control plane (Consul agents) after Consul is upgraded to 1.13.
```

Envoy proxies always use gRPC when communicating to the control plane. This clarifies the situation where a proxy may be disconnected from the control plane.

Reply:
Good catch. I meant that when auto-encrypt and auto-config are in use, I had thought from conversations with engineering that Envoy proxies were unlikely to already be configured to use gRPC **with TLS**. (I accidentally omitted "with TLS" in the original text.)

Reply:
@blake: Should we use stronger language than "in environments where ..." if we think Envoy proxies aren't likely to be configured to use gRPC with TLS in this case? (If you suspect that may not be true, "in environments where" is a good substitute)

Reply:
It depends on how operators have configured Consul and their proxies.

Consul-K8s configures sidecars proxies, ingress, and terminating gateways to connect to the gRPC port over TLS when TLS is enabled in the Helm chart.

The same is not automatically configured for Nomad and VM-based environments. The operator would be required to configure TLS and the appropriate env vars to instruct the proxy to use TLS. If they have not enabled TLS, they will be affected by this bug during upgrade.